doc/extdev/event_callbacks.rst

```diff
@@ -322,7 +322,7 @@ Here is a more detailed list of these events.
 
    .. versionadded:: 1.6
 
-      As a **experimental** event
+      As an **experimental** event
```

Comment:
```suggestion
      As an **experimental** event.
```

Sorry, missed that one!

Reply:
Oh yeh, let's remove the experimental comment; it's been a long time available, and anyway I feel if you are going to introduce experimental things, then you need to emit a warning if they are used, rather than it just being in the documentation where people may not see